core/src/lib/bareos_resource.cc

```diff
@@ -34,7 +34,7 @@ BareosResource::BareosResource()
     , description_(nullptr)
     , rcode_(0)
     , refcnt_(0)
-    , item_present_{0}
+    , item_present_{}
     , inherit_content_{0}
```

Comment:
it is probably a good idea to move these into the class declaration and default this ctor.

Reply:
done